build/ci.go

```diff
@@ -153,6 +153,13 @@ var (
 	// we need to switch over to a recursive builder to jumpt across supported
 	// versions.
 	gobootVersion = "1.19.6"
+
+	// This is the version of execution-spec-tests that we are using.
+	// When updating, you must also update build/checksums.txt.
+	executionSpecTestsVersion = "1.0.2"
+
+	// This is where the tests should be unpacked.
+	executionSpecTestsDir = "tests/spec-tests"
```

Comment:
Interesting addition. @meowsbits, based on your recent checks, do we have anything cooked already in spec-tests?

Reply:
No. But its interesting that the test runner is `execBlockTest` -- https://github.com/etclabscore/core-geth/blob/merge/foundation-release/1.13.1/tests/block_test.go#L75